src/ORM/AssociationCollection.php

```diff
@@ -15,6 +15,7 @@
 namespace Cake\ORM;
 
 use ArrayIterator;
+use Cake\Datasource\AssociationsNormalizerTrait;
```

Comment:
Doesn't this leave us with a lot of duplicated code between ORM and Datasour e packages?